### PR TITLE
Added Announcements page syncing to backend DB.

### LINKED ISSUE #279: [Intertnal]Need a page for announcement from homepage
### Subject of the issue
Need a page for announcement from homepage 

### Your environment
* version of your python
* which browser and its version
* OS Type (iOS/Android/Linux/Windows)

### Steps to reproduce
Tell us how to reproduce this issue. Please provide a working demo as a GIF, if possible.

### Expected behaviour
Tell us what and how exactly is it supposed to  happen

### Actual behaviour
Tell us what happens instead or what went wrong

### PR DESCRIPTION
**Pull request by theryecatcher**
* One suggestion was to have collapsible summaries.

### Issue Reference
This PR addresses the Issue : Fixes #279 

### Summarize
Please, describe what the PR does, the changes you have made.
Added a Table to have the details of the links, created a front end to map the same.

> Also, mention the key points if any to look into while code reviews
Would need to populate the Announcements DB table.
For Example

Emergency Numbers : http://keralanews.gov.in/index.php/main-news/15890-emergency-numbers
Contributions : http://prd.kerala.gov.in/node/21230

### CODE REVIEW

File: templates/home.html [hunk 1]
```diff
@@ -70,6 +70,13 @@ <h1 class="main-logo">kerala<span style="font-weight: normal;">rescue</span></h1
           <span class="ml small">ദുരിതാശ്വാസ ക്യാമ്പുകൾ</span>
         </span>
       </a>
+    <a href="/announcements" class="home-button card" role="button">
+        {% bootstrap_icon "bullhorn" %}
+        <span class="text">
+        Announcements<br/>
+            <span class="ml small">പ്രഖ്യാപനം</span>
```
Comment (mojogeorge) [inline templates/home.html]: I think "അറിയിപ്പുകൾ " would be better Malayalam equivalent

**Comment by shajith** [comment]
@theryecatcher: got this when running the server locally:

```
  Your models have changes that are not yet reflected in a migration, and so won't be applied.
  Run 'manage.py makemigrations' to make new migrations, and then re-run 'manage.py migrate' to apply them.
```

**Comment by theryecatcher** [comment]
The models.py has been changed would need to run the below before testing
```
python manage.py makemigrations
python manage.py migrate
```

**Comment by shajith** [comment]
yup doing that. fwiw, i _think_ we've been generating/committing migrations.

**Comment by theryecatcher** [comment]
I did that in the previous PR and somebody asked me not to. So wasn't sure. :(

File: templates/home.html (hunk 1 above)
Comment (theryecatcher) [reply templates/home.html]: Sure updating it.

**Comment by vrthra** [comment]
Could you please rebase?

**Comment by theryecatcher** [comment]
Rebased and update the repo.

**Comment by johnsamuelwrites** [comment]
The main page has got a new menu item: Relief camps. You need to update the two files: admin.py and views.py

**Comment by theryecatcher** [comment]
Updated the files. Resolving the conflicts. Sorry for the delay was outside.

**Comment by vkaracic** [comment]
Since you're adding a new filter please keep an eye out for https://github.com/IEEEKeralaSection/rescuekerala/pull/425

**Comment by gladiatorvarg** [comment]
I am testing it

**Comment by gladiatorvarg** [comment]
@theryecatcher I have the following suggestions

1. Announcements should be sorted on descending on time added. currently sorted ascending on date.
2. When landing on the page, can you make sure the latest top 10 announcements are displayed.

**Comment by gladiatorvarg** [comment]
It works. can be merged. I have made the above suggestions.

**Comment by shajith** [comment]
@theryecatcher: sorry, mind resolving conflicts again?

**Comment by ultimatecoder** [comment]
We are working on this new requirements. Please merge as soon as possible.

**Pull request merged**